### PR TITLE
[12.0][ADD] purchase_order_uninvoiced_amount: Show uninvoiced amount on purchase order tree

### PR DESCRIPTION
**Pull request by manuelcalerosolis**
Show uninvoiced amount on purchase order tree.

@Tecnativa TT21155

### CODE REVIEW

**Comment by HviorForgeFlow** [comment]
It would be fine to have an autogenerated [readme](https://github.com/OCA/maintainer-tools/tree/master/template/module/readme)

**Comment by manuelcalerosolis** [comment]
Ready to be reviewed @pedrobaeza @ernestotejeda @chienandalu
Please review

File: purchase_order_uninvoiced_amount/models/purchase_order.py
Comment (ernestotejeda) [inline purchase_order_uninvoiced_amount/models/purchase_order.py:20]: Why not simply assign the value in this way?:
```order.amount_uninvoiced = order.currency_id.round(amount_uninvoiced)```

**Review by ernestotejeda — COMMENTED** [review]
Code review: OK
Functionally review (locally): OK

File: purchase_order_uninvoiced_amount/README.rst
Comment (chienandalu) [inline purchase_order_uninvoiced_amount/README.rst]: Generate from oca-gen-readme.

File: purchase_order_uninvoiced_amount/README.rst
Comment (chienandalu) [inline purchase_order_uninvoiced_amount/README.rst]: Format properly, please :)

File: purchase_order_uninvoiced_amount/tests/test_purchase_order_uninvoiced_amount.py
Comment (chienandalu) [inline purchase_order_uninvoiced_amount/tests/test_purchase_order_uninvoiced_amount.py]: Avoid using demo data for tests, as it's prone to unexpected errors and interactions with third modules.

File: purchase_order_uninvoiced_amount/views/purchase_order_view.xml
Comment (chienandalu) [inline purchase_order_uninvoiced_amount/views/purchase_order_view.xml]: s/Univoiced/Uninvoiced

File: purchase_order_uninvoiced_amount/tests/test_purchase_order_uninvoiced_amount.py
Comment (chienandalu) [inline purchase_order_uninvoiced_amount/tests/test_purchase_order_uninvoiced_amount.py]: Don't leave blank lines inside a method :)

File: purchase_order_uninvoiced_amount/tests/test_purchase_order_uninvoiced_amount.py
Comment (chienandalu) [inline purchase_order_uninvoiced_amount/tests/test_purchase_order_uninvoiced_amount.py]: use SavePointCase for better test performance

File: purchase_order_uninvoiced_amount/models/purchase_order.py
Comment (chienandalu) [inline purchase_order_uninvoiced_amount/models/purchase_order.py]: Is it needed to have this stored?

**Review by chienandalu — COMMENTED** [review]
Thanks @manuelcalerosolis Some comments below

File: purchase_order_uninvoiced_amount/models/purchase_order.py
Comment (manuelcalerosolis) [reply purchase_order_uninvoiced_amount/models/purchase_order.py:20]: this is a faster way

File: purchase_order_uninvoiced_amount/README.rst
Comment (manuelcalerosolis) [reply purchase_order_uninvoiced_amount/README.rst]: Done

File: purchase_order_uninvoiced_amount/views/purchase_order_view.xml
Comment (manuelcalerosolis) [reply purchase_order_uninvoiced_amount/views/purchase_order_view.xml]: Fixed

File: purchase_order_uninvoiced_amount/tests/test_purchase_order_uninvoiced_amount.py
Comment (manuelcalerosolis) [reply purchase_order_uninvoiced_amount/tests/test_purchase_order_uninvoiced_amount.py]: Done

File: purchase_order_uninvoiced_amount/README.rst
Comment (manuelcalerosolis) [reply purchase_order_uninvoiced_amount/README.rst]: Done

File: purchase_order_uninvoiced_amount/tests/test_purchase_order_uninvoiced_amount.py
Comment (manuelcalerosolis) [reply purchase_order_uninvoiced_amount/tests/test_purchase_order_uninvoiced_amount.py]: Fixed

File: purchase_order_uninvoiced_amount/tests/test_purchase_order_uninvoiced_amount.py
Comment (manuelcalerosolis) [reply purchase_order_uninvoiced_amount/tests/test_purchase_order_uninvoiced_amount.py]: Done

File: purchase_order_uninvoiced_amount/models/purchase_order.py
Comment (manuelcalerosolis) [reply purchase_order_uninvoiced_amount/models/purchase_order.py]: I think not, but it would have better performance

File: purchase_order_uninvoiced_amount/models/purchase_order.py
Comment (chienandalu) [reply purchase_order_uninvoiced_amount/models/purchase_order.py:20]: I'd say an assignment does precisely an update :)

File: purchase_order_uninvoiced_amount/i18n/purchase_order_uninvoiced_amount.pot
Comment (chienandalu) [inline purchase_order_uninvoiced_amount/i18n/purchase_order_uninvoiced_amount.pot]: maybe you have to regenerate it again ;)

Comment (manuelcalerosolis) [reply purchase_order_uninvoiced_amount/i18n/purchase_order_uninvoiced_amount.pot]: Yes done!

File: purchase_order_uninvoiced_amount/i18n/purchase_order_uninvoiced_amount.pot
Comment (chienandalu) [inline purchase_order_uninvoiced_amount/i18n/purchase_order_uninvoiced_amount.pot:6]: you missed this

Comment (manuelcalerosolis) [reply purchase_order_uninvoiced_amount/i18n/purchase_order_uninvoiced_amount.pot:6]: fixed :-(

**Comment by rousseldenis** [comment]
@manuelcalerosolis There are licensing inconsitencies through files. Could you fix them?

**Review by ernestotejeda — APPROVED** [review]

**Review by chienandalu — CHANGES_REQUESTED** [review]
Before merge, change de commit name, as it's not wip anymore :)

**Review by chienandalu — APPROVED** [review]

**Comment by chienandalu** [comment]
Thanks!

**Review by chienandalu — CHANGES_REQUESTED** [review]
The only thing left here is to change the commit name :)

**Comment by pedrobaeza** [comment]
Main branch was red also. Fixed in 671a2e1e765f91e754ba976c8f1fefbe3eba52c8, so

/ocabot merge nobump

**Pull request merged**
